src/flamegraph/svg.rs

```diff
@@ -140,7 +146,7 @@ var searchcolor = 'rgb(230,0,230)';",
             size: super::FONTSIZE + 5,
             x: (super::IMAGEWIDTH / 2) as f64,
             y: (super::FONTSIZE * 2) as f64,
-            text: "Flame Graph".into(),
+            text: opt.title.clone().into(),
```

Comment:
I don't think you need the `clone()` here? Specifically, I think `&str` should work just fine:
```rust
(&*opt.title).into()
```

Reply:
Done